### PR TITLE
Add note for bug impacting tutorial

### PR DESCRIPTION
**Pull request by keyur32**
Adding a bug to address this customer issue: https://github.com/OfficeDev/Excel-Custom-Functions/issues/87

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[91614aa](https://github.com/OfficeDev/office-js-docs-pr/pull/706/commits/91614aa257a221d13cbf20e7ae56e802bb513654)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[91614aa](https://github.com/OfficeDev/office-js-docs-pr/pull/706/commits/91614aa257a221d13cbf20e7ae56e802bb513654)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[91614aa](https://github.com/OfficeDev/office-js-docs-pr/pull/706/commits/91614aa257a221d13cbf20e7ae56e802bb513654)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/tutorials/excel-tutorial-create-custom-functions.md) | :warning:Warning | | [Details](#user-content-docs%2Ftutorials%2Fexcel-tutorial-create-custom-functions.md)

<a name="docs%2Ftutorials%2Fexcel-tutorial-create-custom-functions.md"></a>
### [docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/tutorials/excel-tutorial-create-custom-functions.md)
 - **Line 147**: **[Warning]** The content part of NOTE/WARNING/CAUTION/IMPORTANT/NEXT syntax is suggested to start in a new line.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201901252036215471-706%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=osfOODAXcSog6E8XUOND9ORHtKYvdR3EENkk%2F5ul8Ok%3D&st=2019-01-25T20%3A32%3A35Z&se=2019-02-25T20%3A37%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tutorials/excel-tutorial-create-custom-functions.md [hunk 1]
```diff
@@ -143,6 +143,8 @@ Integrating data from the Web is a great way to extend Excel through custom func
         // Note: in case of an error, the returned rejected Promise
         //    will be bubbled up to Excel to indicate an error.
     }
+    
+> [!NOTE] In the January Insiders 1901 Build, there is a bug preventing fetch calls from executing which will result in seeing #VALUE!. > To workaround, this please use the [XMLHTTPRequest API](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-runtime#requesting-external-data) to make the web request.
```
Comment (mscharlock) [inline docs/tutorials/excel-tutorial-create-custom-functions.md]:
```suggestion
> [!NOTE] In the January Insiders 1901 build, a known bug prevents `fetch` calls from executing. As a result, cells may instead show #VALUE!. As a workaround, please use the [XMLHTTPRequest API](https://docs.microsoft.com/office/dev/add-ins/excel/custom-functions-runtime#requesting-external-data) to make the web request instead.
```

Comment (mscharlock) [reply docs/tutorials/excel-tutorial-create-custom-functions.md]: Maybe we should put this in the changelog section instead of here? Or in both places?

**Comment by o365devx** [comment]
OPS Build status updates of commit _[9126649](https://github.com/OfficeDev/office-js-docs-pr/pull/706/commits/9126649e8f92edac7d6992211482bf0c121fa79b)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[9126649](https://github.com/OfficeDev/office-js-docs-pr/pull/706/commits/9126649e8f92edac7d6992211482bf0c121fa79b)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[9126649](https://github.com/OfficeDev/office-js-docs-pr/pull/706/commits/9126649e8f92edac7d6992211482bf0c121fa79b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/tutorials/excel-tutorial-create-custom-functions.md) | :warning:Warning | | [Details](#user-content-docs%2Ftutorials%2Fexcel-tutorial-create-custom-functions.md)

<a name="docs%2Ftutorials%2Fexcel-tutorial-create-custom-functions.md"></a>
### [docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/tutorials/excel-tutorial-create-custom-functions.md)
 - **Line 147**: **[Warning]** The content part of NOTE/WARNING/CAUTION/IMPORTANT/NEXT syntax is suggested to start in a new line.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201901252152561172-706%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Y3mYP5qvEhtc8FgymTYcCfBNk5kzy18hDPJvwt7hCsU%3D&st=2019-01-25T21%3A49%3A13Z&se=2019-02-25T21%3A54%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**